app/assets/stylesheets/mixins/_helpers.scss

```diff
@@ -3,8 +3,3 @@
 @function em($px, $base: 19) {
```

Comment:
It doesn't seem like this is used anywhere - perhaps we can just delete the whole helpers file?

Reply:
Removed, thanks.